src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -229,6 +237,10 @@ void getMany(RoutingContext routingContext) {
         multiLoanRecordsResult.after(userRepository::findUsersForLoans))
       .thenCompose(multiLoanRecordsResult ->
         multiLoanRecordsResult.after(loanPolicyRepository::findLoanPoliciesForLoans))
+      .thenCompose(multiLoanRecordsResult ->
+        multiLoanRecordsResult.after(overdueFinePolicyRepository::findOverdueFinePoliciesForLoans))
+      .thenCompose(multiLoanRecordsResult ->
+        multiLoanRecordsResult.after(lostItemPolicyRepository::findLostItemPoliciesForLoans))
```

Comment:
Please make sure that the permission sets have been updated to include permissions for these additional endpoints

Reply:
Changed